src/app/cardsView/base-paginate-card.component.ts

```diff
@@ -24,11 +24,11 @@ export abstract class BasePaginateCardComponent {
     this.loading = true;
     this.cards = source
       .do(res => {
-        this.total = res.count;
```

Comment:
Dlaczego to nie przechodziło na travisie? source jest typu `Observable<PaginateResponse<ParentCard>>`, a `PaginateResponse<>` wygląda tak:
```typescript
export interface PaginateResponse<T extends Card> {
  count: number;
  cards: T[];
}
```